lib/framework.js

```diff
@@ -376,8 +376,8 @@ Framework.prototype.start = function () {
     let proxyUrl = this.options.httpsProxy || null;
     if (proxyUrl) {
       console.log('proxyurl exists', proxyUrl);
-      let httpsProxyAgent = new HttpsProxyAgent(url.parse(proxyUrl));
-      config.defaultMercuryOptions = {
+      let httpsProxyAgent = new HttpsProxyAgent(proxyUrl);
+      config.config.defaultMercuryOptions = {
```

Comment:
Hi @Drastic, without a proxy to test with, I'm happy to take your input that this change needs to happen with the new URL lib.

I get the change on line 379, but I'm not sure I understand what the extra config object is for on line 379?

Reply:
Thanks @Drastic.  I appreciate your work on this as I don't have an env to test this in.  I've currently got [another PR](https://github.com/WebexCommunity/webex-node-bot-framework/pull/95) waiting for review.  I'm going to try to add your commits to that branch so that this can be part of the 2.4.2 release.

Reply:
@jpjpjp to test, it's simple:
1) Set **httpsProxy** to some dummy non-proxy url like 'http://github.com:80'
2) Check if your framework initializes at all
Using Wireshark or *export WEBEX_LOG_LEVEL=warn* you can get more details.

Worth nothing that mercury proxy setting is only for websocket connection. Regular http/https requests require http_proxy or https_proxy environment variable setting.

Reply:
I had some struggle with setting proxy for my deployment, so I had to backtrack it. You can say it has been tested.

Here's webex init as per webex-js-sdk examples:
```js
const Webex = require(`webex`);
const webex = Webex.init({
  credentials: {
    access_token: <your webex access token>
  },
  config: {
    defaultMercuryOptions: {
      agent: httpsProxyAgent
    },
  }
});
```